src/service/dashboards/reports.rs

```diff
@@ -285,9 +296,16 @@ pub async fn enable(
         .map_err(|e| (http::StatusCode::INTERNAL_SERVER_ERROR, e))
 }
 
-impl Report {
+#[async_trait]
+pub trait SendReport {
```

Comment:
I introduced a `SendReport` trait so that we can implement the `send_subscribers` method on `Report`, which is now a foreign type after it was moved out of the `opensobserve` crate and into `config`